packages/component-library/src/_Themes/Brand/BrandTypography.js

```diff
@@ -0,0 +1,15 @@
+// TODO: update to em?
```

Comment:
There's basically nothing standardized here... what do we want to make the law of the land in Typography land?

Reply:
@Dianna Since nobody lives in typography land, you've ascended to the throne by default, so you get to make the laws now 👑

Reply:
👸